seedsource_core/django/seedsource/views.py

```diff
@@ -153,3 +154,22 @@ def get_queryset(self):
 
             point = Point(x, y)
             return get_regions_for_point(point)
+
+
+class ShareURLViewset(viewsets.ModelViewSet):
```

Comment:
Since this viewset only does retrieve and create, rather than extending `ModelViewSet`, which implements everything (including list, delete, etc.), extend the specific classes you need:

```python
class ShareURLViewset(mixins.CreateModelMixin, mixins.RetrieveModelMixin, viewsets.GenericViewSet):
    ...
```

https://www.django-rest-framework.org/api-guide/viewsets/#custom-viewset-base-classes